pkg/kopia/repository/client.go

```diff
@@ -62,10 +62,10 @@ func ConnectToAPIServer(
 
 	opts := &repo.ConnectOptions{
 		CachingOptions: content.CachingOptions{
-			CacheDirectory:            kopia.DefaultClientCacheDirectory,
-			MaxCacheSizeBytes:         int64(contentCacheMB << 20),
-			MaxMetadataCacheSizeBytes: int64(metadataCacheMB << 20),
-			MaxListCacheDuration:      content.DurationSeconds(defaultConnectMaxListCacheDuration.Seconds()),
+			CacheDirectory:         kopia.DefaultClientCacheDirectory,
+			ContentCacheSizeBytes:  int64(contentCacheMB << 20),
```

Comment:
I see different flags(`ContentCacheSizeLimitBytes`, `MetadataCacheSizeLimitBytes`) being used in PR by @kale-amruta : https://github.com/kanisterio/kanister/pull/2393/files#diff-d96603cf822203f32cb69e1031279e1c357193ab6a01bbcf59ce08caa3dc3f8dR66
Just added comment at both of the PRs to confirm which of the flags to be used.

Reply:
https://github.com/kanisterio/kanister/pull/2393#discussion_r1369683114